src/Mod/Fem/femsolver/elmer/equations/linear.py

```diff
@@ -20,11 +20,12 @@
 # *                                                                         *
 # ***************************************************************************
 
-
-__title__ = "_Linear"
+__title__ = "FreeCAD FEM _Linear"
```

Comment:
should I remove the `_` from `_Linear` ?

Reply:
FreeCAD FEM solver elmer equation _Linear